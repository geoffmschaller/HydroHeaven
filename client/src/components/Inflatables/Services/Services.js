import React from 'react';
import styles from './Services.module.css';
import {Link} from "react-router-dom";

const Services = (props) => {

	return (
		<div className={styles.services}>

			<div className={styles.service}>
				<div className={styles.title}>Central Coast Experts.</div>
				<div className={styles.holder}>
					<div className={styles.description}>We are excited to continue and serve the central coast a high quality product and service we stand
						behind.
					</div>
				</div>
				<div className="clear"/>
			</div>
			<div className={styles.service}>
				<div className={styles.title}>Your Chemical Resource</div>
				<div className={styles.holder}>
					<div className={styles.description}>From Chlorine to Test Strips: we stock everything you need to maintain your spa and keep it up and
						running!
					</div>
				</div>
				<div className="clear"/>
			</div>
			<div className={styles.service}>
				<div className={styles.title}>In House Technician</div>
				<div className={styles.holder}>
					<div className={styles.description}>Our full time technician handles the complete range of warranty, maintenance, and repair work. No third
						party hassle!
					</div>
					<Link to={"/service-contact"}>
						<button>Schedule a Service</button>
					</Link>
				</div>
				<div className="clear"/>
			</div>

		</div>
	);

};

export default Services;