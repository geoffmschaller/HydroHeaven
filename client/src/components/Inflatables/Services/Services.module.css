.services {
  width: 100%;
  padding: 0 10px; }
  .services .service {
    width: 100%;
    margin-bottom: 15px; }
    .services .service .title {
      font-size: 16px;
      margin-bottom: 5px;
      text-transform: uppercase; }
    .services .service .holder {
      padding: 0 15px; }
      .services .service .holder .description {
        font-size: 12px;
        color: #808b96;
        margin-bottom: 10px; }
      .services .service .holder button {
        color: #f8c471;
        border: 2px solid #f8c471;
        float: right; }
        .services .service .holder button:hover {
          color: white;
          background-color: #f8c471; }
