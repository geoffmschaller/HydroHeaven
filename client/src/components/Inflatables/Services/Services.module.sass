$button-color: #f8c471
$description-color: #808b96

.services
	width: 100%
	padding: 0 10px

	.service
		width: 100%
		margin-bottom: 15px

		.title
			font-size: 16px
			margin-bottom: 5px
			text-transform: uppercase

		.holder
			padding: 0 15px

			.description
				font-size: 12px
				color: $description-color
				margin-bottom: 10px

			button
				color: $button-color
				border: 2px solid $button-color
				float: right

				&:hover
					color: white
					background-color: $button-color