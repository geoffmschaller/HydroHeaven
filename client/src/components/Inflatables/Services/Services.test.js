import React from 'react';
import Services from "./Services";

const wrapper = <Services/>;

it("Services", () => {
	expect(wrapper).toMatchSnapshot();
});