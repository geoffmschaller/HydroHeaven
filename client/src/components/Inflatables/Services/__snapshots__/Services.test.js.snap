// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Services 1`] = `<Services />`;
